test/test-setup.js

```diff
@@ -0,0 +1,3 @@
+import { jest } from '@jest/globals';
+
+global.jest = jest;
```

Comment:
I kinda ignored this in the other PR but I probably should not have. I don't want this to be considered a standard practice. Globals are a hack, especially with esm modules and are not standard.  Its never good or safe to rely on global "this" mutations.  Its only really meant for some polyfills stuff and this more about hiding things instead of being explicit which is what we want.  Its also kinda why they had to make a package called `@jest/globals` because "magic global injection" is agains the esm code standard. We do not gain anything but lose understanding because now these are different from everything else. Also if you need something else out of `@jest/globals` how are you going to do it? Are you going to further pollute the global scope for it?

Reply:
Do you know what the alternative is?  What should be done instead?

We are still having problems that weren't as easily overcome as we thought this morning.

Reply:
I guess you mean don't do `global.jest = jest` and instead do the `import { jest } from '@jest/globals';` in each individual file that needs it?

Reply:
I reverted the last commit, which added jest as a global. Thanks for this input Jared.